pyrs/peaks/peak_collection.py

```diff
@@ -155,6 +156,43 @@ def set_peak_fitting_values(self, subruns, parameter_values, parameter_errors, f
         self._fit_cost_array = np.copy(fit_costs)
         self.__set_fit_status()
 
+    def get_d_reference(self):
+        return self._d_reference
+
+    def set_d_reference(self, values):
+        """Set d reference values
+
+        Parameters
+        ----------
+        values :
+            1D numpy array or floats
+
+        Returns
+        -------
+
+        """
+        self._d_reference = [values] * self._sub_run_array.size
```

Comment:
Should check if a ndarray was supplied and copy it. It'll be needed at a much later date when the d-reference actually varies with subrun